boomerang/src/lib.rs

```diff
@@ -57,24 +72,24 @@ fn check_canister_ids() {
     );
     assert_eq!(
         Principal::from_text("buwm7-7yaaa-aaaar-qagva-cai").unwrap(),
-        NICP_LEDGER_ID
+        Principal::from_slice(&[0, 0, 0, 0, 2, 48, 1, 170, 1, 1])
     );
     assert_eq!(
         Principal::from_text("jcmow-hyaaa-aaaaq-aadlq-cai").unwrap(),
-        WTN_LEDGER_ID
+        Principal::from_slice(&[0, 0, 0, 0, 2, 0, 0, 215, 1, 1])
     );
     assert_eq!(
         Principal::from_text("tsbvt-pyaaa-aaaar-qafva-cai").unwrap(),
-        WATER_NEURON_ID
+        Principal::from_slice(&[0, 0, 0, 0, 2, 48, 1, 106, 1, 1])
```

Comment:
useless test, you shouldn't remove this test.

Reply:
I agree. I made the changes necessary to make the test useful again